diagnostic.md

```diff
@@ -48,7 +64,8 @@ In your own words, please answer the following question: WHAT IS THE DOM?
 Please give two examples of things that are commonly confused for the DOM.
 
 <!-- your answer starts here -->
-
+1) Looking at the HTML source of a web page is not the same thing as looking at the DOM.
+2)
```

Comment:
Right the HTML source is not the DOM, neither is the page that displays on our screen.